src/shared/components/Inputs/Input.jsx

```diff
@@ -12,12 +12,13 @@ const Input = ({ label, error, onChange, onBlur, value }) => {
     }
 
     return (
-        <FormControl variant="outlined" fullWidth error={!!error}>
+        <FormControl variant="outlined" fullWidth error={!!error} required={required}>
             <InputLabel htmlFor={label}>{startCase(label)}</InputLabel>
             <OutlinedInput
                 id={label}
                 label={label}
                 name={label}
+                type={type}
```

Comment:
tas type=email yra naudojamas netam kad butina defininti kad email, o tam kad tiesiog validuotu message. Tai cia geriau paliekam text ir tada `validationSchemoje` ipisam vietom Yup.string, Yup.email ar kazkas tokio

Reply:
dar del auto input kai buna ant https, tik del to ir dadejau